src/theory/quantifiers/sygus/sygus_module.h

```diff
@@ -49,7 +50,7 @@ class QuantifiersInferenceManager;
  * Modules implement an initialize function, which determines whether the module
  * will take responsibility for the given conjecture.
  */
-class SygusModule
+class SygusModule : protected EnvObj
```

Comment:
It looks like this doesn't actually use anything from `Env`. Do we have to derive here?

Reply:
This inheritance in part allows `Rewriter::rewrite` to be replaced by `rewrite` throughout the majority of the quantifiers code.